tests/plot_sync/test_receiver.py

```diff
@@ -78,7 +78,7 @@ def create_payload(payload_type: Any, start: bool, *args: Any) -> Any:
 def assert_error_response(plot_sync: Receiver, error_code: ErrorCodes) -> None:
     connection = plot_sync.connection()
     assert connection is not None
-    message = connection.last_sent_message
+    message = connection.last_sent_message  # type: ignore[attr-defined]
```

Comment:
I don't understand why you need the mypy suppression here. If there's a good reason, and it needs to stay there, it should probably be a comment in the code

Reply:
Added!